apisyouwonthate.yml

```
@@ -31,18 +31,15 @@ rules:
       field: get
       function: truthy
 
-  # paths-kebab-case:
-  #   description: Should paths be kebab-case.
-  #   message: '{{property}} should be kebab-case (lower case and separated with hyphens)'
-  #   severity: warn
-  #   given: $.paths[*]~
-  #   then:
-  #     function: casing
-  #     functionOptions:
-  #       type: "kebab"
-  #       separator: 
-  #         char: /
-  #         allowLeading: true
+  paths-kebab-case:
+    description: Should paths be kebab-case.
+    message: '{{property}} should be kebab-case (lower case and separated with hyphens)'
+    severity: warn
+    given: $.paths[*]~
+    then:
+      function: pattern
+      functionOptions:
+        match: "^(\/[a-z0-9-.]+|\/{[a-zA-Z0-9_]+})+$"
```

Nice! I'd suggest this one, as it supports trailing slashes or home routes: `/`. 

```suggestion
        match: "^(\/|[a-z0-9-.]+|{[a-zA-Z0-9_]+})+$"
```